crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -360,3 +384,76 @@ impl JsonRpcHandler {
         Ok(nonce)
     }
 }
+
+fn convert_broadcasted_tx(
+    broadcasted_tx: BroadcastedTransaction,
+    chain_id: StarknetChainId,
+) -> RpcResult<starknet_in_rust_tx::Transaction> {
```

Comment:
why didn't use the transaction from starknet-core

Reply:
Resolved in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/99/commits/c3f070b62ff1982a46b8f4ac42c92e2cbb1e6cf3